README.md

```diff
@@ -1191,9 +1191,22 @@ GET /apps/:guid?fields=expensive_field
 }
 ```
 
-### Proposal: Fields For Sub-Resources
+### Fields For Sub-Resources
```

Comment:
My impression of `fields` (both here and in JSON:API) was to return a subset of already returned data. In that case, you could use `include` + `fields` to return sparse versions of the included resources. 

This proposal (in addition to the "Hidden Fields" proposal above) views `fields` as a way to also include additional data with a request.

Reply:
What we want to do is something closer to hidden fields, but for related resources. If `fields` query param is used either for space versions or hidden fields as per this proposal, then `fields[related.resource]` can be the equivalent for either sparse fields or hidden fields in a related resource